src/app/home/home.component.html

```diff
@@ -44,10 +43,11 @@ <h2 class="title caption">Docker Tools and Workflows for the Sciences</h2>
       <div fxLayout="column wrap" fxLayoutAlign="center center" fxLayoutGap="20px">
         <h3 class="search-title">New to Dockstore? Start here</h3>
         <div fxLayout="row wrap" fxLayoutAlign="center center" fxLayoutGap="20px" class="pb-5">
-          <a mat-raised-button href="#developerIntroduction" class="button-link">Developers</a>
-          <a mat-raised-button href="#scientistIntroduction" class="button-link">Scientists</a>
+          <a mat-raised-button href="#developerIntroduction" class="button-link">Create and share</a>
```

Comment:
You'll get different opinions on this, but I don't like "Create and share" and "Discover and launch". If I'm a brand new user, that's still too vague to me, when I'm trying to figure out what Dockstore is. Maybe it's just harder for me, because I don't fall into either category.

I prefer what you had before, but again, I'm probably not the target audience.

Reply:
I'll revert to what I had before, and we can further discuss naming and stuff in the google docs I am making.